pkg/compact/compact.go

```diff
@@ -25,6 +25,7 @@ import (
 	"github.com/thanos-io/objstore"
 	"golang.org/x/sync/errgroup"
 
+	"github.com/thanos-io/thanos/internal/cortex/util/multierror"
```

Comment:
Oh maybe you want to use the multierror from the TSDB package? I am trying to avoid using anything from our vendored cortex now since we are planning to remove them

Reply:
Good point, updated.